packages/truffle-hdwallet-provider/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "truffle-hdwallet-provider",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I think the lerna workflow means we shouldn't update versions in the PR, instead doing it as part of the `lerna publish` process